### DISCUSSION TITLE
Good way to convert manuals/docs into org mode files

### DISCUSSION THREAD

**Discussion by idlip** [jgm/pandoc#10076] [Q&A] [answered]
Hi everyone!

I'm a org-mode user, and after reading docs/manuals in `epub` and `info`, I think org-mode would be a good candidate to store them for me.

So, I'd like to know what would be the better way to convert html/epub files to cleaner org mode files.
By clean I mean, I dont need those `<<XYZ>>` radio or internal targets which serves no purpose of jumping link.

Note: Default converter serves good purpose, I'm looking for some tips and other suggestions on this.

I'd also like to know how others prefer to read docs/manuals or even a novel.

**Comment by jgm** [comment, marked as answer]
You should be able to use a filter to strip off the identifiers from the Header elements in the AST; then you won't get the targets.

Save this as stripids.lua and use `--lua-filter stripids.lua` on the command line.
```lua
function Header(el)
  el.identifier = ""
  return el
end
```

**Reply by idlip** [reply]
Thank you for the quick response!

This removes the property drawer custom_id links (which is good enough)

I will look into targets then, got a hang of filters documentation.

Edit: Div identifier got rid of <<XYZ>> targets!
